data/mods/Aftershock/recipes/deconstruction.json

```diff
@@ -142,6 +142,19 @@
       [ [ "identification_module", 1 ] ]
     ]
   },
+  {
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "afs_radiobeacon",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_TOOLS",
```

Comment:
I believe uncraft recipes do not have categories.

```suggestion
```

Reply:
You are right, thanks for noticing the error